packages/@aws-cdk/aws-lambda/test/inline.expected.json

```diff
@@ -1,106 +0,0 @@
-{
```

Comment:
This is an old/unused integ test expectation file that contained a `// tslint:disable:no-console`